applications/soupbintcp-gateway/src/main/java/com/paritytrading/nassau/soupbintcp/gateway/Events.java

```diff
@@ -80,7 +80,7 @@ public void receive() throws IOException {
 
                         try {
                             receiver.receive();
-                        } catch (IOException e1) {
+                        } catch (IOException e) {
```

Comment:
Perhaps we can clean up also this little change to keep the diff at the minimum? 🤓